scico/denoiser.py

```diff
@@ -5,7 +5,26 @@
 # user license can be found in the 'LICENSE' file distributed with the
 # package.
 
-"""Denoisers."""
+"""
```

Comment:
The user sees this when looking carefully at the source code or with `help(scico.denoiser)`. Do you think this is enough? Perhaps if these packages are installed there could be an additional warning at the time of import?

Reply:
The user should also see this when looking at the online API docs for `scico.denoisers`. Good question as to whether it's enough. A warning on importing the module is not a bad idea.